src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="item" data-qa="card">
+    <img
+      src="./images/imac.jpeg"
+      alt="APPLE A1419 img"
+      class="item__image"
+    >
+    <span class="item__name">
+      Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+    </span>
+    <span class="item__code">
+      Код товара: 195434
+    </span>
+    <div class="item__rates">
+      <div class="stars">
+        <div class="stars__star star1">
+        </div>
```

Comment:
use modifier class, also put a closing div tag on the same line

```suggestion
      <div class="stars">
        <div class="stars__star stars__star--1"></div>
```